CONTRIBUTING.md

```diff
@@ -151,6 +157,9 @@ You can write examples in the documentation if they are simple concepts to demon
 your feature is more complex, consider adding to the examples or tutorials for MetPy.
 
 You can build the documentation locally to see how your changes will look.
+After setting up your [development environment](#setting-up-your-development-environment) above,
+use ``conda install --file ci/doc_requirements.txt`` to install required packages to build our documentation.
```

Comment:
Can we mention that the user must be at the highest-level folder in their clone (likely called "MetPy", unless they changed it), if they aren't already there, before they execute ``conda install --file ci/doc_requirements.txt``?